notary-server/src/domain/auth.rs

```diff
@@ -0,0 +1,10 @@
+use serde::Deserialize;
+
+/// Structure of each whitelisted record
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct AuthorizationWhitelistRecord {
+    pub name: String,
```

Comment:
Ah I thought it would be good to have this field so that we have a record of who owns certain API key in case that intel is needed in the future (e.g. checking if we have given someone API key before, or if we want to remove someone's access) — or you think that is totally unnecessary?

Reply:
Wouldn't that be more troublesome/error prone to maintain two records? Or you see some overheads of tracking name in this single list?

Reply:
Wouldn't that be more troublesome/error prone to maintain two records? Or you see some overheads of tracking name in this single list?